UNRELEASED.md

```diff
@@ -30,6 +30,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 - Added a `alwaysRenderCustomProperties` to `ThemeProvider` for elements that render outside of the DOM tree to their parent context ([#3652](https://github.com/Shopify/polaris-react/pull/3652))
 - Fixed keyboard interactions for the `Tab` component ([#3650](https://github.com/Shopify/polaris-react/pull/3650))
 - Fixed keyboard interaction when selected Tab was focused and rendering the wrong `::before` colour ([#3669](https://github.com/Shopify/polaris-react/pull/3669))
+- Added focus ring to disclosure tab when tabbing with keyboard([#3675](https://github.com/Shopify/polaris-react/pull/3675))
+- Fixed windows high contrast mode on hover within disclosure menu and displaying active state upon click for `::before` ([#3675](https://github.com/Shopify/polaris-react/pull/3675))
```

Comment:
Wasn't sure if I should have squished all these together?

Reply:
If I can't condense my changes into one sentence, it's an indication for me to separate my fixes into two (or more) PRs to make sure they're small and focused.

Totally up to you if you want to do this!

Reply:
OOOH, super helpful. I think I'll stick to it like this this PR, but next time will make it much more condensed! THANK YOU JACKIE!! 💌